### PR TITLE
[4.0] Add RootNamespace tags to projects

### PR DESCRIPTION
**Pull request by DylanFPS**
### Purpose of this PR

Adds the RootNamespace tags to the projects, to stop ReSharper from complaining about bad namespaces.

### Testing status

Not required

### Comments

Next steps: Changing project filenames, but we don't really have to do that anytime soon as it's just a cosmetic thing at this point.

### CODE REVIEW

File: src/OpenTK.Mathematics/OpenTK.Mathematics.csproj [hunk 1]
```diff
@@ -3,6 +3,7 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <Deterministic>true</Deterministic>
+    <RootNamespace>OpenToolkit.Mathematics</RootNamespace>
```
Comment (varon) [inline src/OpenTK.Mathematics/OpenTK.Mathematics.csproj]: Can we call this Math? It's a little less typing...

Comment (DylanFPS) [reply src/OpenTK.Mathematics/OpenTK.Mathematics.csproj]: Changed the namespace

File: src/Generator.Bind/XML/ParsingHelpers.cs [hunk 2]
```diff
@@ -163,7 +163,7 @@ public static TypeSignature ParseTypeSignature([NotNull] string type)
                 var firstPointerLevelIndex = typeName.IndexOf('*');
                 var lastPointerLevelIndex = typeName.LastIndexOf('*');
 
-                pointerLevel = Math.Abs(lastPointerLevelIndex - firstPointerLevelIndex) + 1;
+                pointerLevel = System.Math.Abs(lastPointerLevelIndex - firstPointerLevelIndex) + 1;
```
Comment (Nihlus) [inline src/Generator.Bind/XML/ParsingHelpers.cs]: Is this required? I can't think of a reason the binder uses the Math subsystem.

Comment (DylanFPS) [reply src/Generator.Bind/XML/ParsingHelpers.cs]: Apologies, was the result of find+replace

**Comment by DylanFPS** [comment]
Closing to reflect internal discussions. After all outstanding PRs have been merged, we will proceed to change the project & folder names instead of add extra tags to the csproj's. This will finalize the namespace change.

**Pull request closed without merging**